adds helix and wezterm to their correspondent sections (#15)
flobsx/awesome-alternatives-in-rust: README.md

#### [termite](https://github.com/thestinger/termite)

* [Alacritty](https://github.com/alacritty/alacritty) - A cross-platform, OpenGL terminal emulator.

> * [WezTerm](https://wezfurlong.org/wezterm/index.html) - Powerful cross-platform terminal emulator and multiplexer implemented in Rust.

Author: ```suggestion
* [WezTerm](https://github.com/wez/wezterm) - A GPU-accelerated cross-platform terminal emulator and multiplexer.
```
